app/lib/actions/katello/repository_set/enable_repository.rb

```diff
@@ -2,6 +2,8 @@ module Actions
   module Katello
     module RepositorySet
       class EnableRepository < Actions::EntryAction
+        middleware.use Actions::Middleware::BackendServicesCheck
```

Comment:
I guess every action that's going to talk to pulp deserves to check that the pulp is running